### DISCUSSION TITLE
Spring boot 3.0.0

### DISCUSSION THREAD

**Discussion by JamesERi** [Netflix/dgs-framework#1352] [General]
Hi, is spring boot 3.0.0 on the roadmap?  When can we expect a release?

Current issues I can see in library is that `spring.factory` files need replacing with new import files. Still using `javax.*` libraries instead of `jakarta.*` libraries.  The latter issue prevents me testing current code with spring boot 3 and manually importing configuration.

**Comment by srinivasankavitha** [comment]
Hi - yes this is on our roadmap for the first quarter of next year. There
is also a WIP PR that one of our contributors is working on :
https://github.com/Netflix/dgs-framework/pull/1230. We will likely maintain
a branch with 2.x and 3.x.
